qiskit_ibm/accountprovider.py

```diff
@@ -139,6 +140,7 @@ def _backends(self) -> Dict[str, IBMQBackend]:
         """
         if not self.__backends:
```

Comment:
This statement will be false if the backends are only partially initialized. That is, if I do `provider.get_backend(some_backend)`, it will initializes `self.__backends` with `some_backend`. This method will return only `some_backend` instead of all backends.

Reply:
Updated! The `_backends` property will now pull all backends just once (dependent on `_loaded_all_backends`). This will stay false if all are loaded individually, however. The only way around that would be to know somehow how many backends there are in total without directly querying them.